### ISSUE TITLE
Unable to load xml containing parameterized module for ParameterNumber (IntelliJ 13)

### ISSUE THREAD

**Issue by ckorakidis** [jshiell/checkstyle-idea#134]
If `ParameterNumber` in the xml file is `<module name="ParameterNumber" />`, the file can be imported
If I change it to something like

    <module name="ParameterNumber">
        <property name="max" value="4"/>
        <property name="tokens" value="METHOD_DEF"/>
    </module>

I get `The CheckStyle rules file could not be loaded.</b><br>unable to parse configuration stream - The content of element type "module" must match "(module|property|metadata)*".:169:12`
The content of the row 169 is the last row of the before-mentioned parameterized configuration: 		`</module>`

**Comment by jshiell** [comment]
I fear I can't reproduce this. I've copy & pasted the segment you list above into several of my rules files and it's worked (and flagged errors) correctly in each.

I'm using IDEA 14/OS X 10.10.2/JDK 1.6.0_65, but this shouldn't make a different - that error is being thrown from CheckStyle, which is for some reason unable to parse the file.

My only idea at present is encoding - is the file definitely UTF-8, and there aren't any unprintable characters lying about?

If you can provide your rules file (via Dropbox or something, to ensure encoding and such doesn't alter) I can also have a try at reproducing it with that.

**Comment by ckorakidis** [comment]
I don't know what was the reason it couldn't be imported; the only difference with the rest of the lines was that when I changed the `<module name="ParameterNumber">` and added the properties, the new lines were using tab indentation instead of spaces. When I was removing the additional lines and the parameter number node was `<module name="ParameterNumber"/>`, it could be imported.
I tried it again with the additional configuration as shown in the description above, and it worked this time.
